tests/PeriodTest.php

```diff
@@ -546,7 +562,7 @@ public function expectedPeriodLengths()
             [13, Period::make('2018-01-01', '2018-01-15', null, Boundaries::EXCLUDE_ALL)],
 
             [24, Period::make('2018-01-01 00:00:00', '2018-01-01 23:59:59', Precision::HOUR)],
-            [24, Period::make('2018-01-01 00:00:00', '2018-01-02 00:00:00', Precision::HOUR, Boundaries::EXCLUDE_END)],
```

Comment:
Could you undo these styling changes?

Reply:
Rolled back last commit to undo changes.